sites/friday/build_docs.sh

```diff
@@ -39,7 +39,7 @@ else
 fi
 
 echo "Moving the hugo binary from cache into the application."
-cp "${PLATFORM_CACHE_DIR}/hugo_${HUGOVERSION}/hugo" "${PLATFORM_APP_DIR}"
+cp "${PLATFORM_CACHE_DIR}/hugo_${HUGOVERSION}/hugo" "${PLATFORM_APP_DIR}/sites/friday"
```

Comment:
I wonder if there's a way we can move the hugo binary still into `${PLATFORM_APP_DIR}` but then pass it a parameter for each site? Or maybe we copy the binary into  `${PLATFORM_APP_DIR}` but then create symlinks from each site back to the root binary?

Reply:
seems wasteful to have 2 copies of the same binary